direct_db_manager.py

```
@@ -12,7 +13,11 @@
 import sqlite3
 import json
 import hashlib
+from PyQt5.QtCore import QThread, pyqtSignal
 import resources
```

Ruff (0.12.2)

`resources` imported but unused

Remove unused import: `resources`

(F401)